source/index.html.md

```diff
@@ -209,9 +211,9 @@ page | 1 | Page to display - see "Pagination" section
 per_page | 250 | Items per page - see "Pagination" section
 include_picture | false | If set to true, the result will include the profile picture thumbnail in a field named `main_picture_url_tile`.
 picture_version | null | Can be set to `original`, `large` or `thumb` to change the included picture version. The picture will be included in a field named `picture_url`. _(Only applies if `include_picture` is true)_
-fields | name,gender | Can be used to modify the fields included in the response. Possible values are: `age`, `gender`, `first_name`, `last_name`, `name`, `main_profession`, `professions`, `languages`, `representative`, `updated_at`.
+fields | name,gender | Can be used to modify the fields included in the response. Possible values are: `age`, `gender`, `gender_new`, `gender_searchability`, `first_name`, `last_name`, `name`, `main_profession`, `professions`, `languages`, `representative`, `updated_at`.
 order | id | Changes the order of returned results. Possible values are: `id`, `name`, `last_name`
-gender | null | Allows filtering by gender. Possible values are: `m`, `f`, `i`.
+gender | null | Allows filtering by gender. Possible values are: `m`, `f`, `i`, `male`, `female`, `transgender_female`, `transgender_male`, `non_binary`, `custom`
```

Comment:
For consistency we should introduce a `gender_new` query parameter, and not allow querying any of the new values via the legacy gender param. This will be confusing otherwise compared to the show view.

Reply:
> I'll add the gender_new parameter

:+1: 

> disallow the new values for gender and disallow the short form for gender_new

I'm also okay with not disallowing it if doing so would unnecessarily complicate our backend. Simply not documenting it would be sufficient.